### PR TITLE
[bug 1590541][bug 1202653] Docker image fixes and product-details update

### PR DESCRIPTION
**Pull request by mastizada**
* Change mysql client to default-mysql-client for docker
* Switch from MySQL-python to mysqlclient library
* Update django-mozilla-product-details package
* Set download directory for product_details
* Update product details

### CODE REVIEW

File: env-dist [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
 # Assuming user remo with password remo and db name remo
 # Mozillians
-MOZILLIANS_API_URL = https://mozillians.org/api/v2/
+MOZILLIANS_API_URL=https://mozillians.org/api/v2/
 # Default database setting for docker setup
-DATABASE_URL = 'mysql://remo:remo@db:3306/remo'
+DATABASE_URL='mysql://remo:remo@db:3306/remo'
 # Uncomment the below DATABASE_URL setting if you're running the setup
 # locally using virtualenv
 # DATABASE_URL = 'mysql://remo:remo@localhost:3306/remo'
```
Comment (MichaelKohler) [inline env-dist]: can you do here it as well?

**Comment by MichaelKohler** [comment]
I did some tests locally and didn't notice anything off. The test_view_edit_current_voting test failure is acknowledged as that one failed at https://travis-ci.org/mozilla/remo/jobs/599546390. Can you have a look at the other 3 test failures? Other than that r+ with the minor comment from me.

(obviously I can't say if these changes will have an impact on production though)

**Comment by mastizada** [comment]
@MichaelKohler Other issues are related to email. Outbox is empty after the event comment. Do you know where it is trying to send an email for comments? I tried to find the signal, checked EventComment model and form, and couldn't find the email part.

**Comment by mastizada** [comment]
Issue was that some users (recipients) passed to `send_remo_mail` function were in integer type instead of long, and the task was only accepting email string and long pk. Adding a check for integer with or solved email issues.

File: remo/settings.py [hunk 2]
```diff
@@ -167,6 +167,9 @@
         'visibility_timeout': config('REDIS_VISIBILITY_TIMEOUT', default=604800, cast=int)
     }
 
+# Product Details
+PROD_DETAILS_DIR = Path(ROOT, 'lib/product_details_json')
```
Comment (akatsoulas) [inline remo/settings.py]: Why is this line needed?

Comment (mastizada) [reply remo/settings.py]: This is for downloading product-details files into the lib folder. Running update products command will fetch new files.

**Comment by mastizada** [comment]
@akatsoulas yes, it also was my concern. While fixing the docker issue the product details started crashing, so I updated the package and that update also solved second bug. I can split it into 2 PRs if necessery. As for 2 whitespace commits, I missed the commented line and after Michael pointed it out, that line also changed with the update of travis's env file. At the end, I will squash all of them into 1 commit

**Comment by akatsoulas** [comment]
Closing this PR because the site will be decommissioned soon. Thank you for the time spent on this.
